packages/terra-search-field/src/SearchField.jsx

```diff
@@ -265,6 +265,7 @@ class SearchField extends React.Component {
     const clearButton = this.searchText && !isDisabled
       ? (
         <Button
+          data-terra-search-field-button="Clear"
```

Comment:
While we are in here, we should add `data-terra-search-field-button="Search"` to the search button as well

Reply:
We should also add docs on these data attributes since they were added for testing. Here's an example of how this might look: https://engineering.cerner.com/terra-ui/components/terra-notification-dialog/notification-dialog/notification-dialog#testing

Reply:
Added here https://github.com/cerner/terra-core/pull/3207/commits/3719667d69adc3eef032db7b37867b2ee9c928ed